### ISSUE TITLE
QTooltip shows after containing drawer is closed

### ISSUE THREAD

**Issue by dsl101** [quasarframework/quasar#1093]
### Software version

Quasar: 0.14.7
OS:
Node:
NPM:
Browsers: Mobile Safari
iOS: 10
Android:
Any other software related to your bug:

### What did you get as the error?
A tooltip attached to a button is displayed when the button is hidden

### What were you expecting?
No tooltip

### What steps did you take, to get the error?
The following sample App.vue has a toolbar with a left-hand drawer containing a single 'close' button with a tooltip. On desktop, this works as expected—the tooltip is displayed on mouse-over, and vanishes when the button is clicked. On iOS safari, the tooltip is shown when the close button is clicked, and remains even when the drawer has been hidden.
```
<template>
  <q-layout ref="layout" view="lHr lpr lFr">
    <q-toolbar slot="header">
      <q-btn flat @click="$refs.layout.toggleLeft()">
        <q-icon name="menu" />
      </q-btn>
      <q-toolbar-title>Tooltip Test</q-toolbar-title>
    </q-toolbar>

    <div slot="left">
      <q-toolbar>
        <q-btn flat @click="$refs.layout.hideLeft()">
          <q-icon name="close" />
          <q-tooltip>
            Close
          </q-tooltip>
        </q-btn>
      </q-toolbar>
    </div>
    <router-view />
  </q-layout>

</template>
```

**Comment by cgestes** [comment]
I have the same error with tooltip on buttons in a popover.

When the popover is closed, the tooltips stays.

**Comment by Blfrg** [comment]
+1 on mobile (Android+Chrome Stable / also Ubuntu Desktop+Chrome Stable dev-mode as mobile)
Touch button [in drawer]-->drawer hides-->tooltip shows

**Comment by rstoenescu** [comment]
Hi,

Fixed in v1.0.
